library/src/main/java/com/google/android/exoplayer/extractor/webm/WebmExtractor.java

```diff
@@ -1303,6 +1314,35 @@ public void initializeOutput(ExtractorOutput output, int trackId, long durationU
       this.output.format(format);
     }
 
+    private static Pair<List<byte[]>, Long> parseFourCCPrivate(ParsableByteArray buffer)
+            throws ParserException {
+        try {
+            buffer.setPosition(buffer.getPosition()+16); // size, skip width(4), height(4),
+                                                         // planes(2), and bitcount(2)
+            long biCompression = buffer.readLittleEndianUnsignedInt();
+            buffer.setPosition(buffer.getPosition()+20); // skip sizeimage(4), xpel/meter(4),
+                                                       // ypel/meter(4), clrUsed(4), clrImportant(4)
+            List<byte[]> privateData = new ArrayList<>();
+            final int baseBytesLeft = buffer.bytesLeft();
+            byte[] scratchBytes = new byte[baseBytesLeft];
+            buffer.readBytes(scratchBytes, 0, baseBytesLeft);
+
+            int realStartPos;
+            for(realStartPos = 0; realStartPos < (baseBytesLeft - 4); ++realStartPos) {
```

Comment:
I'll be honest.  After I tried just passing the whole thing (and it went pink) I just went and double checked how Kodi is handling it because they had the same pink issue when they hooked up the hardware decoder when Nvidia added it.  My thought is when in doubt check Kodi.

https://github.com/xbmc/xbmc/blob/7f13ce1ff612283bef6dfc090c017a9e6416ac3a/xbmc/cores/VideoPlayer/DVDCodecs/Video/DVDVideoCodecAndroidMediaCodec.cpp

Koying handles the VC1 extra data at line 453 (AV_CODEC_ID_VC1).  If I had to guess the 0x00,0x00,0x01,0x0f is a profile flag value.  I'm assuming he looked for that and discarded anything in front as the vc1 coded for Nvidia must be looking for the profile value first (another assumption by me).